.gitignore

```diff
@@ -18,4 +18,7 @@ maps/*.png
 *.swp
 
 # OS X leftovers
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# imported scripts from manasource
+scripts/lua
```

Comment:
I don't know why names didn't show up on the text I copied in my previous comment but I said the first line and @Ablu blu said the second.

Reply:
Yeah, I got that (and I saw the chatlog). But indeed I would expect running "make install" after compiling manaserv to fix this as well, since it should already put the scripts in a path that is searched, so that you don't need to copy them into the client data.

Reply:
Maybe I am misunderstanding what you mean but all this change is doing is having git ignore the files that get copied over from manasource. It sounds like you are talking about the problem I was having with the files not getting copied over in the first place. If I did something wrong and they should have been copied over automatically then the file I changed that effects that is on the wiki: "https://github.com/tales/sourceoftales/wiki/How-to-setup-a-local-server". I added a line to the setup script there that automatically copies the scripts for you. I don't even know of that setup script works though.

Reply:
I mean running "make install" after compiling `manaserv` does the following:
```
bjorn@baldur:~/manaserv$ sudo make install
[sudo] password for bjorn: 
[  9%] Built target enet
[ 44%] Built target manaserv-account
[100%] Built target manaserv-game
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/share/manaserv/scripts/lua/libmana-constants.lua
-- Installing: /usr/local/share/manaserv/scripts/lua/libmana.lua
-- Installing: /usr/local/share/manaserv/scripts/lua/npclib.lua
-- Installing: /usr/local/bin/manaserv-account
-- Set runtime path of "/usr/local/bin/manaserv-account" to ""
-- Installing: /usr/local/bin/manaserv-game
-- Set runtime path of "/usr/local/bin/manaserv-game" to ""
```
And the path "/usr/local/share/manaserv" is added to the search path in the server binary:
```
src/common/resourcemanager.cpp:    PHYSFS_addToSearchPath(PKG_DATADIR, 1);
```
With `PKG_DATADIR` specified in the CMake file:
```
CMakeLists.txt:    SET(PKG_DATADIR ${CMAKE_INSTALL_PREFIX}/share/manaserv)
```
So, if `manaserv` is "installed", then there should be no need to copy over those script files, and then there is no need to ignore them.

But I see the instructions on that wiki page you linked to not mention running `make install` after compiling manaserv. Either that should be fixed, or there should be a way to run manaserv without needing the install step, but copying over and ignoring the scripts is I think a somewhat poor hack that I'd rather not have as the documented way of doing things.

Reply:
Thank you for clarifying that for me. I have tested that and it seems to check out so I have removed my additions to the .gitignore  file and edited the script on the wiki to remove what I had put and instead run "make install". This does seem better to me even though it is less standalone. If you all decide on something else then I can change it again.